packages/lumx-react/src/components/link-preview/LinkPreview.tsx

```diff
@@ -25,6 +25,8 @@ interface LinkPreviewProps extends GenericProps {
     theme?: Theme;
     /** Thumbnail image source */
     thumbnail?: string;
+    /** Active thumbnail cross origin. */
+    isThumbnailCrossOriginEnabled?: boolean;
```

Comment:
I think it would be better to do:
```tsx
thumbnailProps?: ThumbnailProps;
```
And spread the props in the `<Thumbnail>`
So you could use the LinkPreview like this:
`<LinkPreviewProps thumbnailProps={{isCrossOriginEnabled: true}} />`
What do you think?

Reply:
@gcornut I thought about it, but that would mean the user could override all of the thumbnail props and therefor alter the `LinkPreview` final design ?
If that's not an issue then yeah totally !

Reply:
@gcornut Changed according to our PM conversation :)